base/cvd/cuttlefish/host/libs/config/defaults/defaults.cc

```diff
@@ -33,7 +34,13 @@ Defaults::Defaults(std::map<std::string, std::string> defaults) {
 }
 
 std::optional<std::string_view> Defaults::Value(std::string_view k) const {
-  return defaults_.at(k);
+  absl::btree_map<std::string, std::string>::const_iterator it =
```

Comment:
nit: Iterators are one of the exceptions to the suggestion to avoid using `auto` in https://google.github.io/styleguide/cppguide.html#Type_deduction.